tools/check/check.go

```diff
@@ -16,50 +16,101 @@
 
+	configProto = flag.String("config", "",
+		("A path to a serialized check.Config protobuf. Any individual field flags will" +
+			" overwrite the message's associated field. Default unmarshalled as binary. Paths" +
+			" ending in .textproto will be unmarshalled as prototext."))
 	quiet   = flag.Bool("quiet", false, "If true, writes nothing the stdout or stderr. Success is exit code 0, failure exit code 1.")
 	verbose = flag.Bool("verbose", false, "Enable verbose logging.")
 
-	checkcrl        = flag.Bool("check_crl", defaultCheckCrl, "Check the CRL for revoked certificates. -get_collateral must be true.")
-	getcollateral   = flag.Bool("get_collateral", defaultGetCollateral, "Get the Collateral for additional checks.")
+	qevendoridS    = flag.String("qe_vendor_id", "", "The expected QE_VENDOR_ID field as a hex string. Must encode 16 bytes. Unchecked if unset.")
+	qevendorid     = cmdline.Bytes("-qe_vendor_id", abi.QeVendorIDSize, qevendoridS)
+	mrseamS        = flag.String("mr_seam", "", "The expected MR_SEAM field as a hex string. Must encode 48 bytes. Unchecked if unset.")
+	mrseam         = cmdline.Bytes("-mr_seam", abi.MrSeamSize, mrseamS)
+	tdattributesS  = flag.String("td_attributes", "", "The expected TD_ATTRIBUTES field as a hex string. Must encode 8 bytes. Unchecked if unset.")
+	tdattributes   = cmdline.Bytes("-td_attributes", abi.TdAttributesSize, tdattributesS)
+	xfamS          = flag.String("xfam", "", "The expected XFAM field as a hex string. Must encode 8 bytes. Unchecked if unset.")
+	xfam           = cmdline.Bytes("-xfam", abi.XfamSize, xfamS)
+	mrtdS          = flag.String("mr_td", "", "The expected MR_TD field as a hex string. Must encode 48 bytes. Unchecked if unset.")
+	mrtd           = cmdline.Bytes("-mr_td", abi.MrTdSize, mrtdS)
+	mrconfigidS    = flag.String("mr_config_id", "", "The expected MR_CONFIG_ID field as a hex string. Must encode 48 bytes. Unchecked if unset.")
+	mrconfigid     = cmdline.Bytes("-mr_config_id", abi.MrConfigIDSize, mrconfigidS)
+	mrownerS       = flag.String("mr_owner", "", "The expected MR_OWNER field as a hex string. Must encode 48 bytes. Unchecked if unset.")
+	mrowner        = cmdline.Bytes("-mr_owner", abi.MrOwnerSize, mrownerS)
+	mrownerconfigS = flag.String("mr_owner_config", "", "The expected MR_OWNER_CONFIG field as a hex string. Must encode 48 bytes. Unchecked if unset.")
+	mrownerconfig  = cmdline.Bytes("-mr_owner_config", abi.MrOwnerConfigSize, mrownerconfigS)
+	reportdataS    = flag.String("report_data", "", "The expected REPORT_DATA field as a hex string. Must encode 64 bytes. Unchecked if unset.")
+	reportdata     = cmdline.Bytes("-report_data", abi.ReportDataSize, reportdataS)
+	minteetcbsvnS  = flag.String("minimum_tee_tcb_svn", "", "The minimum acceptable value for TEE_TCB_SVN field as a hex string. Must encode 16 bytes. Unchecked if unset.")
+	minteetcbsvn   = cmdline.Bytes("-minimum_tee_tcb_svn", abi.TeeTcbSvnSize, minteetcbsvnS)
+
+	rtmrs = flag.String("rtmrs", "",
```

Comment:
Why comma-separated for this list of strings when cabundles uses ":"? Can cabundles use comma separation instead?

Reply:
Done. Used comma separation for cabundles.